travis/unit_tests/INFO.md

```diff
@@ -1 +1,34 @@
-The directory `travis/unit_tests/` contains the unit tests.
+The directory `travis/unit_tests/` contains the **unit tests** for
+AtChem2. These tests check the output of individual subroutines and
+functions of the model.
+
+Modules tested:
+
+- argparse.f90:
+  - split_string
+  - check_name_value_pair_validity
+
+- atmosphereFunctions.f90:
+  - calcAirDensity
+  - calcAtmosphere
+  - convertRHtoH2O
+  - zero_equal (?)
```

Comment:
Not sure what this test does

Reply:
I will leave for now. This PR is in preparation to a new testsuite i have been tinkering with (lots of long flights:)). So it can be dealt with there.